source/dialog/components/LayoutMain.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Container = styled.div`
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+`;
+
+class LayoutMain extends Component {
+    render() {
+        return <Container>{this.props.children}</Container>;
+    }
+}
+
```

Comment:
This component can be simplified to:

```js
export const LayoutMain = styled.div`...`;
```

it doesn't need to be a class. (React prefers state-less components when possible, like this one)

Reply:
What about the children? Why won't someone please think of the `this.props.children`??
![image](https://user-images.githubusercontent.com/3869469/35063405-abb0448c-fbcf-11e7-8533-b70f2a4ea6ae.png)

Reply:
Hahahaha, it just works. Automatically (its done in the styled-components. Renders the children as is

Reply:
I tried it, it didn't work. Heaps of errors unfortunately.

Reply:
Lets get this through then, we can look at that later 👍🤟